modules/markup/markdown/meta.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2020 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package markdown
+
+import (
+	"strings"
+
+	"gopkg.in/yaml.v2"
+)
+
+func isYAMLSeparator(line string) bool {
+	line = strings.TrimSpace(line)
+	for i := 0; i < len(line); i++ {
+		if line[i] != '-' {
+			return false
+		}
+	}
+	return len(line) != 0
```

Comment:
```suggestion
	return len(line) > 2
```

May this is better. I am thinking about this:

```yml
name: "Bug :bug:"
labels:
  -
    bug
  - 
    "help wanted"
```

`-` will act like separator

Reply:
Done